dxr/plugins/python/indexers.py

```diff
@@ -21,6 +21,8 @@
             'py_function': QUALIFIED_NEEDLE,
             'py_derived': QUALIFIED_NEEDLE,
             'py_bases': QUALIFIED_NEEDLE,
+            'py_callers': QUALIFIED_NEEDLE,
+            'py_called_by': QUALIFIED_NEEDLE,
```

Comment:
It looks like I quit supporting called-by queries in clang awhile back. There was some implementation motivation, and I couldn't think of a use case for them. Can you?